app/utilities/metadata_parser_v2.py

```diff
@@ -102,9 +103,13 @@ class RunnerMetadataSchema(Schema, StripWhitespaceMixin):
     @validates_schema
     def validate_schema_name_is_set(self, data, **kwargs):
         # pylint: disable=no-self-use, unused-argument
-        if data and not (data.get("schema_name") or data.get("schema_url")):
+        if data and not (
+            data.get("schema_name")
+            or data.get("schema_url")
+            or data.get("cir_instrument_id")
+        ):
             raise ValidationError(
-                "Neither schema_name or schema_url has been set in metadata"
+                "Neither schema_name, schema_url or cir_instrument_id has been set in metadata"
             )
```

Comment:
We should update this to the check is that one and only one of them is provided.

Reply:
Good shout, done 👍